src/main/kotlin/nmcp/NmcpAggregation.kt

```diff
@@ -48,6 +48,13 @@ abstract class NmcpAggregation @Inject constructor(
      */
     abstract val verifyStatus: Property<Boolean>
 
+    /**
+     * Timeout for verification (in seconds).
+     *
+     * Default: 600 (10 minutes).
```

Comment:
Wdyt of assigning default property values in `init { publicationWaitTimeout.convention(...) }`?

Then it could be one-time configuration and all the usages would get it if needed.